rdm/init_files/documents/software_plan.md

```diff
@@ -380,15 +404,15 @@ Merge the approved git branch into the `master` git branch, correct any merge co
 
 The final integration prior to a release must formally record the test output in a test record.  The test record must include:
 
-- The list of tests that passed or failed
-- Verification that the results meet the pass/fail criteria listed in the Test Plan [[62304:5.7.4.c]]
-- The version of the software being tested (e.g., the git commit hash)
-- The name of the person who ran the tests [[62304:5.6.6, 62304:5.6.7, 62304:5.7.5, and 62304:9.8]].
+- The list of tests that passed or failed [[62304:5.6.7.a 62304:5.7.5.a]]
+- Verification that the results meet the pass/fail criteria listed in the Test Plan [[62304:5.7.4.c 62304:5.6.7.a]]
+- The version of the software being tested (e.g., the git commit hash) [[62304:5.6.7.b 62304:5.7.5.b]]
+- The name of the person who ran the tests [[62304:5.6.6, 62304:5.6.7, 62304:5.7.5.c, 62304:5.6.7.c and 62304:9.8]].
 
-Any test failures found during the formal release system testing shall be recorded as problem reports.  See the [prepare problem report activity](#prepare-problem-report) for details [[62304:5.7.2]].  If any change requests are implemented in response to these problem reports, the tests must be re-run.  If it is deemed unnecessary to re-run some of the tests, the justification as to why shall be included in the test record [[62304:5.7.3 note that the risk management activities for (c) will be handled as part of the unit implementation and testing activity]].
+Any test failures found during the formal release system testing shall be recorded as problem reports [[62304:5.6.8 62304:5.7.4.d]].  See the [prepare problem report activity](#prepare-problem-report) for details [[62304:5.7.2]].  If any change requests are implemented in response to these problem reports, the tests must be re-run [[62304:5.7.3.a 62304:5.7.3.b]].  If it is deemed unnecessary to re-run some of the tests, the justification as to why shall be included in the test record [[62304:5.7.3.c note that the risk management activities for (c) will be handled as part of the unit implementation and testing activity]].
 
 **Output:** Test record and problem reports
-
+[[62304:5.6.3 62304:5.6.4 62304:7.3.3.d]]
```

Comment:
Perhaps we should use `[[62304:5.6.3, 62304:5.6.4, and 62304:7.3.3.d]]`?

Reply:
Fixed.